core/src/main/res/values/strings.xml

```diff
@@ -642,6 +642,7 @@
     <string name="gpodnetsync_error_descr">An error occurred during syncing:\u0020</string>
     <string name="gpodnetsync_pref_report_successful">Successful</string>
     <string name="gpodnetsync_pref_report_failed">Failed</string>
+    <string name="gpodnetsync_username_characters_error">Username contains character(s) not accepted.</string>
```

Comment:
I did not add the notice to explicitly state that `mail != user` because a mail is not the only thing that is not allowed. I did visit `https://gpodder.net/register/` and found out a only usernames that contain only letters, digits , hyphens and underscores are accepted. So that why i decided to use the message. I believe `mail != user` would be misleading. Any addition insights on this will be helpful. Thanks.

Reply:
Maybe we can use a similar text that gpodder itself uses: `Usernames may only contain letters, digits, hyphens and underscores`. I think that sounds better.

Reply:
Let me make the change.

Reply:
I have made the necessary changes here.